solidity/test/bridge/Bridge.test.ts

```diff
@@ -1659,16 +1818,63 @@ describe("Bridge", () => {
                   context(
                     "when there is a pending request for the given redemption key",
                     () => {
+                      before(async () => {
+                        await createSnapshot()
+
+                        // Make a request targeting the given wallet and
+                        // redeemer output script. Tested request will use
+                        // the same parameters.
+                        await makeRedemptionAllowance(
+                          thirdParty,
+                          mainUtxo.txOutputValue
+                        )
+                        await bridge
+                          .connect(thirdParty)
+                          .requestRedemption(
+                            walletPubKeyHash,
+                            mainUtxo,
+                            redeemerOutputScriptP2WPKH,
+                            mainUtxo.txOutputValue
+                          )
+                      })
+
+                      after(async () => {
+                        await restoreSnapshot()
+                      })
+
                       it("should revert", async () => {
-                        // TODO: Implementation.
+                        await expect(
+                          bridge
+                            .connect(thirdParty)
+                            .requestRedemption(
+                              walletPubKeyHash,
+                              mainUtxo,
+                              redeemerOutputScriptP2WPKH,
+                              requestedAmount
+                            )
+                        ).to.be.revertedWith(
+                          "There is a pending redemption request from this wallet to the same address"
+                        )
                       })
                     }
                   )
                 })
 
                 context("when amount is below the dust threshold", () => {
                   it("should revert", async () => {
-                    // TODO: Implementation.
+                    // Initial dust threshold set in the tests `fixture`
+                    // for tests is 100000. A value lower by 1 sat should
+                    // trigger the tested condition.
+                    await expect(
+                      bridge
+                        .connect(thirdParty)
+                        .requestRedemption(
+                          walletPubKeyHash,
+                          mainUtxo,
+                          redeemerOutputScriptP2WPKH,
+                          99999
```

Comment:
We could export constants from the fixture, [just like we do in ecdsa tests](https://github.com/keep-network/keep-core/blob/b0acf0e54eb7d82c6ab339d0d09f0c542c2649ea/solidity/ecdsa/test/fixtures/index.ts#L19-L31). Then, we'd import constants from here and reference them.